src/test/java/org/datadog/jmxfetch/TestServiceChecks.java

```diff
@@ -83,7 +83,7 @@ public void testServiceCheckWarning() throws Exception {
         assertEquals(Reporter.formatServiceCheckPrefix("too_many_metrics"), scName);
         // We should have an OK service check status when too many metrics are getting sent
         assertEquals(Status.STATUS_OK, scStatus);
-        assertEquals(scTags.length, 1);
+        assertEquals(scTags.length, 3);
         assertTrue(Arrays.asList(scTags).contains("instance:jmx_test_instance"));
```

Comment:
Yes, I can do that. In fact it was a good idea. The way I was adding tags made these tags:

```yml
env: stage
newTag: test
```
be output as just `stage` and `test` I've added another commit that will change that to `env:stage` `newTag:test` which is what I assume we want.